### ISSUE TITLE
feat: expand prefer-web-first-assertions, or new rule for `isXyz` methods.

### ISSUE THREAD

**Issue by MattyBalaam** [playwright-community/eslint-plugin-playwright#356]
Consider these scenarios:

```
    await page.getByRole("button").isDisabled();

    expect(page.getByRole("button").isVisible()).toBeTruthy();
```

If you hover over the `isX` methods there is some documentation that states:

> NOTE If you need to assert that an element is disabled, prefer [expect(locator).toBeDisabled([options])](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) to avoid flakiness. See [assertions guide](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) for more details.

`prefer-web-first-assertions` does not catch these I have not been able to find a lint rule to enforce this, maybe I have missed it?

**Comment by MattyBalaam** [comment]
I see that this lint rule now fixes the second case, but is there a way to handle

` await page.getByRole("button").isDisabled()`

So that it changes to 

`    await expect( component.getByRole("button")).toBeDisabled()`

I realise that is possibly a separate case

**Comment by mskelton** [comment]
@MattyBalaam IMO that's outside the scope of the lint rule. Why would users write code like that anyway?

**Comment by MattyBalaam** [comment]
This is to ensure they are using await(expect… to test for a condition
